support/cas-server-support-pac4j/src/main/java/org/apereo/cas/support/pac4j/config/support/authentication/Pac4jAuthenticationEventExecutionPlanConfiguration.java

```diff
@@ -125,6 +125,7 @@ private void configureFoursquareClient(final Collection<BaseClient> properties)
     private void configureGoogleClient(final Collection<BaseClient> properties) {
         final Pac4jProperties.Google google = casProperties.getAuthn().getPac4j().getGoogle();
         final Google2Client client = new Google2Client(google.getId(), google.getSecret());
+        client.setName("Google");
```

Comment:
Thanks for the comment - what's the rationale behind making the name configurable?  If it's anything but "Google" (perhaps GoogleClient would be better) then the btn-google class from bootstrap-social won't match (when using the default fragment).

This is due to the line:
`final String name = client.getName().replaceAll("Client\\d*", StringUtils.EMPTY);`
in `DelegateClientAuthenticationAction`, where that name goes on to be used as both the name and type in a `ProviderLoginPageConfiguration`